vignettes/aggregation.Rmd

```diff
@@ -177,7 +177,7 @@ Explicit imputation for missingness (zero-filling in our case) can be important
 for protecting against bugs in all sorts of downstream tasks. For example, even
 something as simple as a 7-day trailing average is complicated by missingness.
 The function `epi_slide()` looks for all rows within a window of 7 days anchored
-on the right at the reference time point (when `n = 7` and `align = "right"`).
+on the right at the reference time point (when `after = 6`).
```

Comment:
Think this should be `before = 6`.  Anchored on the right at the reference time point == right-aligned == before is n-1 and after is 0

Reply:
"left" and "right" are starting to get really confusing for me right now.  Maybe we should favor "trailing" and "leading", or "right-aligned/trailing", "left-aligned/leading"

Reply:
I fixed the first comment.

As for the second comment, I did the latter.